### DISCUSSION TITLE
When can we expect support for Android?

### DISCUSSION THREAD

**Discussion by Morsmalleo** [tauri-apps/tauri#4198] [Q&A] [answered]
G'day guys, just a quick question about when we can expect Android Support for Tauri? Thanks 🙂

**Comment by FabianLars** [comment, marked as answer]
To my knowledge, we still plan to have the implementation ready-ish late summer at the earliest (excluding the audit and release).

**Reply by nothingismagick** [reply]
Today.
See this merge:
https://github.com/tauri-apps/tauri/pull/5787/commits/df2e027dd97b9cab7cbe350b40bde9bea31d05fe

**Comment by firstdorsal** [comment]
Any updates on this?

**Reply by FabianLars** [reply]
See #843, #3884 and the #mobile channel on discord. Please don't post a "any updates?" questions there, i will just mark them as off-topic anyway - if there are noteworthy updates we will post them.

**Comment by xeoshow** [comment]


**Reply by JonasKruckenberg** [reply]
Please don't spam, asking in one place is more than enough given that there are no news. If you're on our discord server or following Tauri on any social media platform you will get the news as soon as we have them. Until then assume no news just means no news.

**Reply by xeoshow** [reply]
Sorry for this, just deleted it ...

**Comment by wusyong** [comment]
If you really want to try bleeding edge features, mobile support has already landed in `next` branch. 
However, this branch contain several breaking changes. So mobile support won't comes to 1.x release.
We plan to release 2.0 alpha along with `next` branch development just like we did in pre 1.0 release.
We should publish alpha in next month. But before that you can always follow this [note](https://hackmd.io/@lucasfernog/BkHI4e18j) for current instructions.
